torch/_dynamo/convert_frame.py

```diff
@@ -340,7 +340,7 @@ def _convert_frame_assert(
             return None
 
         if is_generator(code):
-            unimplemented("generator")
+            unimplemented("generator", "U0001")
```

Comment:
I'm a bit worried if we end up at with every unique error code for every graph break, then probably it's the same as using plaintext message to aggregate. E.g., I'm [searching ```unimplemented```](https://github.com/search?q=repo%3Apytorch%2Fpytorch+unimplemented+path%3A%2F%5Etorch%5C%2F_dynamo%5C%2F%2F&type=code&p=1) under ```torch/_dynamo``` and found almost every graph break has a different reason. Even there is some one is similar, I think it's challenging to maintain an error codes table for developers to follow.

Reply:
So I think there are benefits, and they outweigh the cost of maintaining the error codes table:

* Some unimplemented functions do string interpolation in their message. An error code allows you to common these up into one error code
* An error code guarantees that each code site has a unique string; with free-form strings, there could be false conflicts where two sites happen to be named the same way
* If someone rewords an unimplemented message for clarity, that will break aggregation. The error code can stay stable across renaming
* The error code allows for the possibility of an out of band wiki describing all error codes, how they're triggered, and how to work around them

The main cost of maintaining the error codes is making sure each site has a unique one, which can lead to merge conflicts. There isn't much else you have to do under this scheme.